### ISSUE TITLE
pl.replace as in pd.replace to accept dict mapping

### ISSUE THREAD

**Issue by Arengard** [pola-rs/polars#1554]

**Comment by ritchie46** [comment]
Yes, check the user guide: https://pola-rs.github.io/polars-book/user-guide/howcani/apply/udfs.html

or SO:  https://stackoverflow.com/questions/69575496/how-to-use-groupby-and-apply-with-polars
